src/test/store/markers.test.js

```diff
@@ -330,10 +333,21 @@ describe('Marker schema filtering', function() {
       ['payload no schema', 0, null, { type: 'no schema marker' }],
       ['RefreshDriverTick', 0, null, { type: 'Text', name: 'RefreshDriverTick' }],
       ['UserTiming',        5, 6,    { type: 'UserTiming', name: 'name', entryType: 'mark' }],
+      // The following is a tracing marker without a schema attached, this was a
+      // regression reported in Bug 1678698.
+      ['CompositeToTarget', 7, 8,    { type: 'tracing', category: 'Paint' }],
       ...getNetworkMarkers(),
     ]);
     const { getState } = storeWithProfile(profile);
     const getMarker = selectedThreadSelectors.getMarkerGetter(getState());
+    const markerSchemaByName = getMarkerSchemaByName(getState());
+
+    if (markerSchemaByName.CompositeToTarget) {
```

Comment:
See above, I believe this condition should check `markerSchemaByName.Paint`.

Reply:
Correct.